### PR TITLE
Doc edits of the “Getting started” section.

### PR DESCRIPTION
**Pull request by jasongrout**

### CODE REVIEW

File: docs/source/getting_started/overview.md [hunk 1]
```diff
@@ -6,30 +6,25 @@ JupyterLab goes beyond the classic Jupyter Notebook by providing a flexible and
 extensible web application with a set of reusable components. You can arrange
 multiple notebooks, text editors, terminals, output areas, and custom components
 using tabs/panels and collapsible sidebars. These components are carefully
-designed to enable you to use them together (can send code from a file to a code
-console with a keystroke) or on their own (move cells around a notebook using
-drag-and-drop) to support novel data-driven workflows.
+designed to enable you to use them together or on their own to support new
+workflows (for example, you can send code from a file to a code console with a
```
Comment (jzf2101) [inline docs/source/getting_started/overview.md]: The way this is written, could we also split this into two sentences?

File: docs/source/getting_started/overview.md [hunk 2]
```diff
@@ -6,30 +6,25 @@ JupyterLab goes beyond the classic Jupyter Notebook by providing a flexible and
...
-
-* Use the same [server](https://jupyter-notebook.readthedocs.io/en/stable/) as
-  the classic Jupyter Notebook.
-* Use the same [notebook document
-  format](http://nbformat.readthedocs.io/en/latest/) as the classic Jupyter
-  Notebook.
+JupyterLab also offers a unified model for viewing and handling data formats.
+This allows data in many formats (images, CSV, JSON, Markdown, PDF, Vega,
+Vega-Lite, etc.) to be opened as a file or returned by a kernel as rich output.
+See [File and output formats]() for more information.
```
Comment (jzf2101) [inline docs/source/getting_started/overview.md]: I believe this link is blank?  Should we point to both the user-facing and the developer-facing documentation?

File: docs/source/getting_started/overview.md [hunk 3]
```diff
@@ -6,30 +6,25 @@ JupyterLab goes beyond the classic Jupyter Notebook by providing a flexible and
...
+designed to enable you to use them together or on their own to support new
+workflows (for example, you can send code from a file to a code console with a
+keystroke, or move cells around a notebook or between notebooks with
+drag-and-drop).
 
 **JupyterLab has full support for Jupyter Notebook documents.** In addition, it
-offers other models of interactive computing:
+enables other models of interactive computing, such as:
 
 * [Code Consoles]() provide transient scratchpads for running code
```
Comment (jzf2101) [inline docs/source/getting_started/overview.md]: Link is blank

File: docs/source/getting_started/overview.md [hunk 4]
```diff
@@ -6,30 +6,25 @@ JupyterLab goes beyond the classic Jupyter Notebook by providing a flexible and
...
   interactively, with full support for rich output.
-* [Kernel backed documents]() allow code in any text file (markdown, python, R,
+* [Kernel-backed documents]() allow code in any text file (Markdown, Python, R,
```
Comment (jzf2101) [inline docs/source/getting_started/overview.md]: Link is also blank

**Comment by jzf2101** [comment]
We should probably include in this section information about shortcuts (eg how similar or different they are from those of the Notebook) and vim/emacs

**Comment by jasongrout** [comment]
Let's take links to a separate issue, since there are decisions we need to make around how we are doing those.

**Comment by ian-r-rose** [comment]
Is this ready do go as well?

**Pull request merged**